Add src/duct_hierarchy.edn to :resource-paths

kwrooijen opened the pull request:
## Problem
When building an uberjar all `duct_hierarchy.edn` files are merged to create a full hierarchy. This works fine until you create an uberjar. Your main project's `duct_hierarchy.edn` is not included it the merged hierarchy.

## Solution
Add `src/duct_hierarchy.edn` to the uberjar's profile resource-paths. This will make sure it is included when generating the uberjar.

weavejester commented:
This seems like a bug in Leiningen if true. I'll see if I can reproduce this issue.

kwrooijen commented:
@weavejester My colleagues and I came across this issue again. I think it's because the default duct template has no `:source-paths` definition, and possibly defaults to `["src"]` ? We added our own: `:source-paths ["src/clj" "src/cljs"]` and it broke. We fixed it after changing this to: `["src/clj" "src/cljs" "src"]`.
